test/Microsoft.AspNet.Mvc.Razor.Test/TagHelpers/HiddenFormTagHelperTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Razor.TagHelpers;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Mvc.ViewFeatures;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using Moq;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Razor.Test.TagHelpers
+{
+    public class HiddenFormTagHelperTest
```

Comment:
Instead test with a `CommonTestEncoder` instance. Should confirm double-encoding doesn't happen in unit tests.